masterlist.yaml

```diff
@@ -5218,20 +4897,14 @@ plugins:
         name: 'Beyond Skyrim: Bruma SE on Bethesda.net'
     group: *addonsGroup
     msg:
-      - <<: *hasPluginPatch
-        subs:
-          - 'Convenient Horses'
-          - '[Bruma patch for Convenient Horses](https://www.nexusmods.com/skyrimspecialedition/mods/13812/)'
+      - <<: *patchProvided
```

Comment:
The patch isn't provided at <https://www.nexusmods.com/skyrimspecialedition/mods/10917/>...